.gitignore

```diff
@@ -16,3 +16,4 @@ packages
 .directory
 Thumbs.db
 *.userprefs
+*.ffs_db
```

Comment:
Good point; had a look and yes it does seem to be `sync.ffs_db` in all the places i use it.
So, do i revert this commit, make the change to full file name and make new commit? 
or is there a nicer way to achieve the same?